src/js/lib/content/src/templates.ts

```diff
@@ -0,0 +1,245 @@
+/// This module defines helper methods for templates view.
+
+import { ProjectManager } from './project_manager'
+
+const PM = ProjectManager.default()
+
+const PROJECTS_LIST = 'projects-list'
+const PROJECTS_LIST_NEW_PROJECT = 'projects-list-new-project'
+
+const CARD_SPREADSHEETS = 'card-spreadsheets'
+const CARD_GEO = 'card-geo'
+const CARD_VISUALIZE = 'card-visualize'
+const CARD_BMW_DRIVERS = 'card-bmw-drivers'
+
+const ALL_CARDS = [
+    CARD_SPREADSHEETS,
+    CARD_GEO,
+    CARD_VISUALIZE,
+    CARD_BMW_DRIVERS
+]
+
+/**
+ * The sore for hidden elements.
+ *
+ * When the templates view is loaded, it hides some top-level elements, as
+ * their style is messing up scrolling. Hidden elements will be restored before
+ * loading the IDE.
+ */
```

Comment:
This documentation doesnt tell what is really happening. What are "some elements"? Why they are "messing up scrolling"? Why the scrolling is broken? This comment doesn't describe what is really happening and why this is needed. Also, based on this comment I feel this is kind of a hack. If it is, this should be fixed. Let's talk about it tomorrow.